core/src/net/sf/openrocket/util/TestRockets.java

```diff
@@ -1,7 +1,7 @@
 package net.sf.openrocket.util;
 
 import java.util.ArrayList;
-import java.io.FileOutputStream;
+// import java.io.FileOutputStream;
```

Comment:
why not just remove?

Reply:
I had to figure out how to dump a rocket so I could look at it all over again, and I left that code in the file but commented out because I couldn't really think of a good place to put it as documentation.  If that code is uncommented by somebody adding a new test case, the import will need to be as well (ignorance of github here -- how do I reference some lines of code in my comments, as Billy did above)?
Better solution would be to document it outside the file.

Reply:
Yea, we really should have a common way to debug that.... I've run into that use case a few times, as well. :P 

Well, github's documentation explains it better than I can: 
https://help.github.com/en/github/managing-your-work-on-github/creating-a-permanent-link-to-a-code-snippet

Also, I generally prefer to simply remove commented loc, but if you want to leave it, I'll defer in this case.